### DISCUSSION TITLE
How to be notified of every created thread/discussion?

### DISCUSSION THREAD

**Discussion by SquirrelDeveloper** [SquirrelCorporation/SquirrelServersManager#49] [Q&A]
Does anyone know how, with a organization setup, my user can be notified of every discussion created here? I can't seem to be notified by email

**Comment by chopeta** [comment]
I don't know about organization setup, but don't you have this option inside each project?

![image](https://github.com/SquirrelCorporation/SquirrelServersManager/assets/7503721/0c195c2e-eb90-4f92-a3ba-0e259e12e052)

I get notified about everything :-)

Awaiting for next releases to test.

Thanks for making SSM open-source. Appreciate it.

**Comment by SquirrelDeveloper** [comment]
Yeah, it seems to work that way, thanks for the tip
